src/index.ts

```diff
@@ -118,9 +109,9 @@ export interface Metadata {
  */
 export interface AccessRequest {
   apiVersion: DSRVersion
-  kind: Kind.AccessRequest
+  kind: Kind
   metadata: Metadata
-  requestBody: {
+  request: {
```

Comment:
Several of the request and response fields should be changed to optional (`documents?` For example).

Reply:
Added `?` for optional fields as per the ketch-forwarder documentation.